packages/psss/src/api/queries/useAlerts.js

```diff
@@ -25,3 +26,12 @@ export const useAlerts = (period, orgUnitCodes, alertCategory) => {
     data,
   };
 };
+
+export const useArchiveAlert = alertId =>
```

Comment:
To my knowledge, `get` requests are currently defined under the `queries` folder, and all  `post/put/delete` under `mutations.js`. Unless if there are any "hidden" mutations I'm not aware of, it seems that `useMutation` is not used in `queries`.

I think I like what you did here: grouping queries by resource (eg `alerts`) rather than by operation (eg `get`/`post`). It still feels tempting to make things consistent one way or the other, but I'll leave this decision to you